sources/packages/web/src/components/common/SearchStudents.vue

```diff
@@ -1,60 +1,66 @@
 <template>
   <v-form ref="searchStudentsForm">
     <content-group class="mb-8">
-      <v-row
-        ><v-col>
-          <v-text-field
-            label="Application number"
-            density="compact"
-            data-cy="appNumber"
-            variant="outlined"
-            v-model="appNumber"
-            @keyup.enter="searchStudents"
-            hide-details
-          />
+      <v-row>
+        <v-col cols="11" lg="11" md="10" sm="12">
+          <v-row
+            ><v-col cols="12" lg="3" md="6">
```

Comment:
actually a total of `md ` is more than 12. I think since you gave `lg`, it's working for you. I believe either one is enough.
since we are using `md` everywhere else and to keep it consistent. maybe we can go for `md`. also fo line 5